app/helpers/application_helper.rb

```diff
@@ -6,4 +6,8 @@ def website_url(base_path)
   def content_tagger_url
     Plek.new.external_url_for('content-tagger')
   end
+
+  def preview_url(slug)
+    "#{Plek.new.external_url_for('collections')}/#{slug}"
```

Comment:
I think you intend to link to the equivalent of: https://draft-origin.publishing.service.gov.uk/learn-to-drive-a-car?